finagle-zipkin-core/src/main/scala/com/twitter/finagle/zipkin/core/RawZipkinTracer.scala

```diff
@@ -15,7 +15,7 @@ import java.nio.ByteBuffer
  * @param statsReceiver We generate stats to keep track of traces sent, failures and so on
  * @param timer A Timer used for timing out spans in the [[DeadlineSpanMap]]
  */
-abstract private[twitter] class RawZipkinTracer(
```

Comment:
can you add a comment somewhere here that explains why it must be public so we don't make it private again?